### DISCUSSION TITLE
Inconsistency in Image Size with Laravel Media Library's Responsive Images

### DISCUSSION THREAD

**Discussion by daalvand** [spatie/laravel-medialibrary#3185] [Q&A]
Two Issues with Laravel Media Library

## First

The first issue is related to the size of the images. I have observed that my original image, with a size of **`111kb`** and dimensions of **`1200x695`**, has a larger size (**`178kb`**) in responsive images with the same dimensions.
Even when the responsive image's dimensions are smaller (**`839x486`**), the size is still larger (**`122kb`**).

Here is the original image information:
![image](https://user-images.githubusercontent.com/23280949/218049782-18890f55-43c3-4dac-bc35-7fce5bbd821e.png)
![image](https://user-images.githubusercontent.com/23280949/218047073-dae7f72d-3834-4a9c-b727-0e5818313a28.png)

Here is responsive images information:
![image](https://user-images.githubusercontent.com/23280949/218060506-4639df6d-e119-438a-ab6e-bc2296c4db0e.png)
![image](https://user-images.githubusercontent.com/23280949/218048679-a4d8a3c0-ec08-45c2-89c1-294a0e853460.png)

This issue with the size makes it better to use the original image instead of the responsive one (actually, responsive images are useless). 
Is there a way to resolve this issue?

## second:

The second issue I have encountered is that I observe that two images with the same dimensions, one with one responsive image and the other with six responsive images (is it depend on image size?).

Here is an example:
first one has one responsive image:
![image](https://user-images.githubusercontent.com/23280949/218054193-9638c90d-b8bc-4528-ab24-610f41769076.png)
second one has six responsive images:
![image](https://user-images.githubusercontent.com/23280949/218054588-df01a403-d9ac-400d-87ab-3857e515d1e3.png)

**Comment by fredsal** [comment]
Hi, feel free to make a PR to fix that

**Comment by parallels999** [comment]
Maybe duplicate of #2582
Official answer https://github.com/spatie/laravel-medialibrary/issues/2582#issuecomment-917927690
